frontend/src/main/pages/AdminJobsPage.js

```diff
@@ -43,24 +46,6 @@ const AdminJobsPage = () => {
             [],
             { refetchInterval: refreshJobsIntervalMilliseconds }
         );
-    // Stryker enable  all 
-
-    const jobLaunchers = [
-        {
-            name: "Update Cow Health",
-            form: <JobComingSoon />
```

Comment:
Just replace this line with 
```suggestion
form: <UpdateCowHealthJobForm submitAction={submitCowUpdateHealthJob} />
```

NOT testJob!